content/Mobile_Dev/WebAuthn/WebAuthn_Primer.adoc

```diff
@@ -0,0 +1,54 @@
+
+Roaming authenticators, also known as cross-platform or external authenticators, such as a hardware security key. These authenticators can be used with a laptop or mobile device, etc., and are thus cross-platform. This characteristic enables their use to establish a secure source for verifying the user’s identity (the "root of trust") and for delegating trust to specific devices in the user’s control. The user can therefore authorize other devices as needed ("bootstrap" them). Should one of those other devices be lost or become corrupted, the roaming authenticator is used to authorize a new device. For these reasons, it is a roaming authenticator such as a security key that should be the primary authenticator.
+
+==== What is a trusted device?
+In the realm of traditional authentication the concept of Trusted Devices was introduced as a form of 2FA to make the login experience easier for a user who frequents a device. This often included the storage of a device identifier as a cookie, in cache, or in local storage. This presents an attack avenue, if an attacker gains access to a user's computer, or finds a way to spoof the machine, then all they require is a compromised password to gain access to a user's account. A Platform Authenticator can act as a Trusted Device by relying on the WebAuthn credential already built into the device. For an attacker it’s not just as easy as having access to the users device, they will also need knowledge of the users PIN, or access to the users biometrics (Fingerprint for Android Biometrics, Face for Face ID, etc.)
+
+==== Hardware backed vs copyable credentials
+Some technology providers have begun to implement copyable credentials within their operating systems/ecosystems, oftentimes this is branded as passkeys. You should be aware of the difference between the two credential types so that you can guide your authentication strategy.
+
+Hardware backed credentials are credentials where the private key never leaves the piece of hardware that they were originally created on. This greatly minimizes the risk of your credential being exposed to attackers as they would need the physical device in order to access any of your accounts. These credentials allow for the highest degree of security.
+
+Copayable credentials are credentials that can be leveraged from one device to another. These credentials typically reside in a shared/cloud environment where an authenticated user is able to access all of their credentials from a single location. A user may also be able to pass their credentials to other users for shared access. While these credentials offer a high degree of usability, there is often a huge trade off in the degree of security offered.
```

Comment:
* Consider working in the keyword `low assurance`.
* Consider removing `huge` and a descriptor as it is too vague imho.
* Consider comparing the security of a copyable passkey to social logins.
* Consider rewording the share sentence. like: A user may choose to share their credential with someone who may in turn choose to share that credential with someone else. Any financial or PII data associated with the account will be accessible by everyone who has the credential.

Reply:
Changed in newest commit - Let me know your feedback for this one. Copyable passkeys now take up two paragraphs , but I think the information is relevant